src/core/server/base.js

```diff
@@ -258,6 +264,22 @@ function baseServer(
     });
   }
 
+  // Redirect category pages.
+  app.get(
```

Comment:
Note that I didn't know how to write a test for this, but I did test it manually. I notice that we don't have any tests for the two `302` redirects in this file, so perhaps there isn't a straightforward way to test this?

Reply:
We should have test cases for that, at least for what has been added here.